### ISSUE TITLE
Apprenticeship learning from ADAM for initialization

### ISSUE THREAD

**Issue by bigaidream** [bigaidream-projects/qan#29]
Learning policies from scratch is wasteful. Similar to [Deep Apprenticeship Learning for Playing Video Games], we can initialize the policies of DQNs from ADAM. 

That is, we use ADAM to train a CNN and record the learning rates at every iteration. Then the initial policy of the DQN is trained by mapping the states to learning rates (i.e. a regression). 

Well, of course, we can simply use DQNs to refine the ADAM schedules. But by using apprenticeship learning, we can distill policies from many different learning rate schedulers (e.g. ADAM, AdaDelta) and then use `actor-mimic` to distill different policies into one DQN. 

The reference code can be found here: https://github.com/eparisotto/ActorMimic

**Comment by bigaidream** [comment]
In fact, distilling multiple hand-designed learning rate schedulers can be seen as a new learning rate scheduler, which is not a reinforcement learning algorithm anymore.

**Comment by bigaidream** [comment]
Since the learning rates given by ADAM are on per-feature basis, we might need to average those learning rates at the same layer. 

Alternatively, we can use multi-agent DQNs. But we need to change the state features from weights to something like [Learning step size controllers for robust neural network training].

**Comment by bigaidream** [comment]
Actually, we can simply use DQN to improve the policies of ADAM. It seems that apprenticeship learning is not that necessary.

**Comment by bigaidream** [comment]
Not that attractive.
